x-pack/plugins/synthetics/public/legacy_uptime/state/effects/synthetic_journey_blocks.ts

```diff
@@ -7,16 +7,18 @@
 
 import { Action } from 'redux-actions';
 import { call, fork, put, select, takeEvery, throttle } from 'redux-saga/effects';
-import { ScreenshotBlockDoc } from '../../../../common/runtime_types/ping/synthetics';
+import {
+  ScreenshotBlockDoc,
+  ScreenshotBlockCache,
+  isPendingBlock,
```

Comment:
Why is a helper function being imported from runtime_types?

Reply:
Considering `isPendingBlock` is a type guard, runtime_types seems a suitable common place. (Also, `isScreenshotBlockDoc`, another related type guard, was already exported in there)